src/test/groovy/hudson/plugins/gradle/GradlePluginIntegrationTest.groovy

```diff
@@ -271,6 +272,11 @@ task hello << { println 'Hello' }"""))
         installation.name == 'myGradle'
         installation.home == '/tmp/foo'
 
+        def envVars = new EnvVars()
+        installation.buildEnvVars(envVars)
+        assert envVars.containsKey("PATH+GRADLE")
+        assert envVars.get("PATH+GRADLE") == installation.home + "/bin"
```

Comment:
Downside to that is that we'd have to bump the Gradle plugin's core dependency to 2.7.1 from its current 1.642.1.